sdk/endpoint/ipam.go

```diff
@@ -57,8 +58,13 @@ func (ice *IpamEndpoint) Request(ctx context.Context, request *networkservice.Ne
 		return nil, err
 	}
 
-	//TODO: We need to somehow support IPv6.
-	srcIP, dstIP, prefixes, err := ice.prefixPool.Extract(request.Connection.Id, connectioncontext.IpFamily_IPV4, request.Connection.Context.ExtraPrefixRequest...)
+	/* Quick way to determine whether the pool is IPv4 or IPv6 */
+	currentIPFamily := connectioncontext.IpFamily_IPV4
+	if strings.Contains(ice.prefixPool.GetPrefixes()[0], ":") {
```

Comment:
That shoudl be OK, but to be even more sure, you should count the `:` and have at least 2 of them for a valid IPv6.

Reply:
Resolved 👍